config/config.exs

```diff
@@ -28,3 +28,6 @@ config :phoenix, :json_library, Jason
 # Import environment specific config. This must remain at the bottom
 # of this file so it overrides the configuration defined above.
 import_config "#{Mix.env()}.exs"
+
+# https://hexdocs.pm/joken/introduction.html#usage
+config :joken, default_signer: System.get_env("SECRET_KEY_BASE")
```

Comment:
this needs to match the `SECRET_KEY_BASE` on dashboard.heroku.com/apps/dwylmail/settings for email to work.